.github/workflows/release-management.yaml

```diff
@@ -0,0 +1,196 @@
+          
+          \`odo\` is your playground!
+          
+          Read the developer reference guide on [contributing to odo](/docs/development/contribution) to know more.
+  
+          EOF
+
+          cat "$blogFileName"
+          echo "blogFileName=${blogFileName}" >> $GITHUB_OUTPUT
+
+      - name: Upload Release Notes Artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: release-blog-post-draft
+          path: ${{ steps.generate_draft_blog_post.outputs.blogFileName }}
+          retention-days: 30
+
+      - name: Update version in installation docs
+        run: |
+          prevTag=$(git describe --abbrev=0 --tags $(git rev-list --tags --skip=1 --max-count=1))
+          prevTagWithoutPrefix=${prevTag#v}
+          newTagWithoutPrefix=${TAG_NAME#v}
+          sed -i 's/'${prevTagWithoutPrefix}'/'${newTagWithoutPrefix}'/g' docs/website/docs/overview/installation.md
+
+      - name: Set Git identity
+        run: |
+          git config --global user.email "$GITHUB_ACTOR@users.noreply.github.com"
+          git config --global user.name "$GITHUB_ACTOR"
+          git remote set-url origin https://x-access-token:${{ secrets.GITHUB_TOKEN }}@github.com/${{ github.repository }}
+
+      - name: Create and push branch
+        id: push_branch
+        run: |
+          git switch -c blog_post_${TAG_NAME}
+          git push -u origin blog_post_${TAG_NAME} 
+          if git status --porcelain | grep build/VERSION; then
+            git add build/VERSION
+            git commit -m "Set version in build/VERSION file"
+          fi
+          if git status --porcelain | grep docs/website/docs/; then
+            git add docs/website/docs/
+            git commit -m "Bump version in installation docs"
+          fi
+          if git status --porcelain | grep docs/website/blog/; then
+            git add docs/website/blog/
+            git commit -m "Add release blog post"
+          fi
+          unpushed=$(git cherry -v)
+          if [ -n "$unpushed" ]; then
+            git push
+            echo "pushed=true" >> $GITHUB_OUTPUT
+          else
+            echo "pushed=false" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Create draft PR
+        if: ${{ steps.push_branch.outputs.pushed == 'true' }}
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          cat <<EOF | gh pr create --draft \
+            --title "[WIP] Release PR for ${TAG_NAME}" \
+            --reviewer "$GITHUB_ACTOR" \
+            --base main \
+            --repo ${GITHUB_REPOSITORY} \
+            --body-file -
+          
+          /area documentation
+          /area release-eng
+          /hold
+          
+          Draft Release PR for new release: $TAG_NAME
+
+          Fixes #
+
+          Feel free to edit and push to this branch.
+          
+          NOTE: Remove the hold and consider merging this PR only when binaries for this release are publicly available.
+
```

Comment:
Same, can we have this template in a separate file?

Reply:
Done - see [`58d2a97` (#6403)](https://github.com/redhat-developer/odo/pull/6403/commits/58d2a97b9c844d0caa963df54a53b08e342cc606)